tests/core/fused_gromov_wasserstein_test.py

```diff
@@ -0,0 +1,249 @@
+# coding=utf-8
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for the Fused Gromov Wasserstein."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+import jax
+import jax.numpy as jnp
+import jax.test_util
+from ott.core import gromov_wasserstein
+from ott.geometry import geometry
+from ott.geometry import pointcloud
+
+
+class GromovWassersteinTest(jax.test_util.JaxTestCase):
```

Comment:
maybe this class should be called FusedGromovWassersteinTEst

Reply:
True, thank you, added that to the follow-up pull request.